extensions/wikia/Search/WikiaSearchController.class.php

```diff
@@ -797,7 +797,7 @@ protected function templateExists( $name ) {
 	 * @see SearchControllerTest::testIsCorporateWiki
 	 */
 	protected function  isCorporateWiki() {
```

Comment:
Hey, I'm just wondering if the name of the method is still valid since it doesn't tell if the wiki is corporate anymore :)